views/layouts/layout-example.njk

```diff
@@ -6,18 +6,21 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{{ title }}</title>
     <link href="/stylesheets/main.css" rel="stylesheet" media="all" />
-    <!--[if lt IE 9]><script src="/javascripts/ie.js"></script><![endif]-->
-
     {#- Include any additional stylesheets specified in the example frontmatter #}
     {% for stylesheet in stylesheets %}
     <link href="{{ stylesheet }}" rel="stylesheet" media="all" />
     {%- endfor %}
 
-    <script src="/javascripts/application.js"></script>
+    <!--[if lt IE 9]>
+      <script src="/javascripts/ie.js"></script>
+    <![endif]-->
+    <script src="/javascripts/vendor/modernizr.js"></script>
+    {% include "_tracking-head.njk" %}
```

Comment:
Is this meant to be included as part of this change?

Reply:
I can make a separate PR for that. Looks like we didn't add tracking on the example pages, which I thought might be useful.

Reply:
Suggest doing as a separate PR